jekyll/_cci1/language-nodejs.md

```diff
@@ -65,12 +65,11 @@ test:
 ```
 
 ### Deployment
-
 CircleCI offers first-class support for [deployment]( {{ site.baseurl }}/1.0/configuration/#deployment).
-When a build is green, CircleCI will deploy your project as directed
-in your `circle.yml` file.
-We can deploy to Nodejitsu and other PaaSes as well as to
-physical servers under your control.
+When a build is green,
+CircleCI will deploy your project as directed in your `circle.yml` file.
+We can deploy to other PaaSes as well as physical servers you manage.
 
-If you have any trouble, please [contact us](https://support.circleci.com/hc/en-us)
+If you have any trouble,
+please [contact us](https://support.circleci.com/hc/en-us),
```

Comment:
please don't say please :) I assume there is some wording in the 2.0 docs for this that you can copy